isis/src/base/apps/hist/main.cpp

```diff
@@ -236,3 +246,41 @@ void IsisMain() {
   delete hist;
   p.EndProcess();
 }
+
+
+// Helper function to fill in the auto calculated min/max.
+void helperButtonCalcMinMax() {
+
+  UserInterface &ui = Application::GetUserInterface();
+
+  // Setup a cube for gathering stats from the user requested band
+  QString file = ui.GetFileName("FROM");
+
+  Cube inCube;
+  CubeAttributeInput attrib = ui.GetInputAttribute("FROM");
+  if (attrib.bands().size() != 0) {
+     vector<QString> bands = attrib.bands();
+     inCube.setVirtualBands(bands);
+  }
+
+  inCube.open(file, "r");
+
+  LineManager line(inCube);
+  Statistics cubeStats;
+
+  for (int i = 1; i <= inCube.lineCount(); i++) {
+    line.SetLine(i);
+    inCube.read(line);
+    cubeStats.AddData(line.DoubleBuffer(), line.size());
+  }  
```

Comment:
Is there a reason why you manually did this instead of using the Cube::statistics function?

Reply:
The cube stats and hist don't work inside the GuiHelper environment. Not sure why, thought I'd investigate later.

Reply:
Strange and kind of concerning, but not an issue for this PR.